metals/src/main/scala/scala/meta/internal/metals/debug/ExtendedScalaMainClass.scala

```diff
@@ -42,10 +44,21 @@ object ExtendedScalaMainClass {
       arguments: List[String],
       mainClass: String,
   ): String = {
-    val jvmOptsString = jvmOptions.mkString(" ")
+    val jvmOptsString =
+      if (jvmOptions.nonEmpty) jvmOptions.mkString("\"", "\" \"", "\"") else ""
     val classpathString = classpath.mkString(File.pathSeparator)
     val argumentsString = arguments.mkString(" ")
-    s"$javaHome $jvmOptsString -classpath $classpathString $mainClass $argumentsString"
+    // We need to add "" to account for whitespace and also escaped \ before "
+    val escapedJavaHome = javaHome.toNIO.getRoot().toString +
+      javaHome.toNIO
+        .iterator()
+        .asScala
+        .map(p => "\"" + p + "\"")
```

Comment:
Is this easier to read?
```suggestion
        .map(p => s""""$p"""")
```
Ugh I hate these lol